.editorconfig

```diff
@@ -9,5 +9,19 @@ charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 
+[*.ts]
+indent_style = tab
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
All but `indent_style` here is the same as for `*` - I think that means you can just omit them entirely.